website/source/docs/guides/connect-gateways.md

```diff
@@ -0,0 +1,348 @@
+---
+layout: "docs"
+page_title: "Connecting Services Across Datacenters"
+sidebar_current: "docs-guides-connect-gateways"
+description: |-
+  Connect services and secure inter-service communication across datacenters
+  using Consul Connect and mesh gateways.
+---
+## Introduction
+
+Consul Connect is Consul’s service mesh offering, which allows users to observe
+and secure service-to-service communication. Because Connect implements mutual
+TLS between services, it also provides users with a way to help services in
+different datacenters communicate with each other through mesh gateways. Using
+gateways for inter-datacenter communication can prevent each Connect proxy from
+needing an accessible address, and frees operators from worrying about IP
+overlap between datacenters.
+
+In this guide, you will configure Consul Connect in multiple, joined Consul
+datacenters and use gateways to facilitate inter-service traffic between them.
```

Comment:
Everywhere in this document we should probably refer to "gateways" as "mesh gateways". Right now maybe it doesn't matter but if/when we also support API gateways then we will need to differentiate them.

Reply:
adding a note in the introduction about this. Adding mesh everywhere makes some sentences confusing because it makes nouns really long